src/common/helpers.rs

```diff
@@ -39,7 +39,6 @@ pub fn create_search_runtime(max_search_threads: usize) -> io::Result<Runtime> {
 
     runtime::Builder::new_multi_thread()
         .worker_threads(search_threads)
-        .enable_time()
```

Comment:
Do you know if there is any Drawback of using `enable_time`?

Reply:
In this case, no drawbacks other than it being obsolete, as it is also enabled with `enable_all`.

Or do you mean what the drawback may be of it enabling at all?

Reply:
I mean, is there an overhead for this option or something? Why don't we just enable it everywhere

Reply:
Since this doesn't affect this PR itself I'll merge now.